packages/main-ui/src/layouts/i18n.rs

```diff
@@ -12,4 +12,49 @@ translate! {
         ko: "로그인",
         en: "Login",
     },
+    
+    welcome: {
+        ko: "환영합니다!",
+        en: "Welcome!",
+    },
+
+    congraturation: {
+        ko: "‘서비스명’에 오신 것을 환영합니다!\n익명성과 신뢰를 바탕으로 안전한 투표 환경을 제공합니다.",
+        en: "Welcome to 'Service Name'!\nWe provide a safe voting environment based on anonymity and trust.",
+    },
+
+    start_poll: {
+        ko: "투표 시작하기",
+        en: "Start voting",
+    },
+
+    enter_nickname: {
+        ko: "닉네임을 입력해주세요",
+        en: "Please enter your nickname",
+    },
+
+    special_characters: {
+        ko: "특수문자는 입력할 수 없습니다.",
+        en: "Special characters are not allowed.",
+    },
+
+    agree_email: {
+        ko: "[필수]이메일 및 계정주소 수집에 동의합니다.",
+        en: "[Required]I agree to collect email and account address.",
+    },
+
+    continue_with_google: {
+        ko: "Google로 계속하기",
+        en: "Continue with Google",
+    },
+
+    quick_sign_in: {
+        ko: "간편 로그인",
+        en: "Quick Sign-in",
+    },
+
+    next: {
+        ko: "다음",
+        en: "Next",
+    },
```

Comment:
Use separated translate as below

```suggestion
translate! {
    CongratuationTranslate,
    
    welcome: {
        ko: "환영합니다!",
        en: "Welcome!",
    },

    congraturation: {
        ko: "‘서비스명’에 오신 것을 환영합니다!\n익명성과 신뢰를 바탕으로 안전한 투표 환경을 제공합니다.",
        en: "Welcome to 'Service Name'!\nWe provide a safe voting environment based on anonymity and trust.",
    },

    start_poll: {
        ko: "투표 시작하기",
        en: "Start voting",
    },

    enter_nickname: {
        ko: "닉네임을 입력해주세요",
        en: "Please enter your nickname",
    },

    special_characters: {
        ko: "특수문자는 입력할 수 없습니다.",
        en: "Special characters are not allowed.",
    },

    agree_email: {
        ko: "[필수]이메일 및 계정주소 수집에 동의합니다.",
        en: "[Required]I agree to collect email and account address.",
    },

    continue_with_google: {
        ko: "Google로 계속하기",
        en: "Continue with Google",
    },

    quick_sign_in: {
        ko: "간편 로그인",
        en: "Quick Sign-in",
    },

    next: {
        ko: "다음",
        en: "Next",
    },
}
```